messages/run-command.md

```diff
@@ -1,39 +1,90 @@
+
+If you've created a custom configuration file that overrides the default rule and engine properties, you can specify it with the --config-file flag.
+
+By default, the command outputs the results in the terminal. Use the --output-file flag to print the results to a file; the format of the results depends on the extension you provide. For example, "--output-file results.csv" creates a comma-separated values file. You can specify one of these extensions:
+
+- .csv
+- .html or .htm
+- .json
+- .junit or .junit.xml
+- .sarif or .sarif.json
+- .xml 
 
 # command.examples
 
-*DRAFT*: Command examples
+- Analyze the files in the current folder using the recommended rules; use concise table output:
+
+    <%= config.bin %> <%= command.id %>
+
+- Analyze the files in the current folder using all available rules, but show details of the violations:
+
+    <%= config.bin %> <%= command.id %> --rule-selector all --view detail
+
+- Analyze only two files, and fail if the run encounters a violation of with severity level of "high" or higher:
+
+    <%= config.bin %> <%= command.id %> --workspace ./my-workspace/init.ts --workspace ./my-workspace/dosomething.ts --severity-threshold high
+
+- Specify a custom configuration file and output the results to a file in CSV format, with severity threshold of moderate (3) :
+
+    <%= config.bin %> <%= command.id %> --config-file ./code-analyzer.yml --output-file results.csv --severity-threshold 3
+
+- Analyze the files in the specified workspace, but run only ESLint rules that have a moderate severity (3) and the default RetireJS rules with any severity:
+
+    <%= config.bin %> <%= command.id %> --workspace ./my-workspace --rule-selector eslint:3 --rule-selector retire-js:default
+
+- Analyze the files in the current folder, but run only the "getter-return" ESLint rule and the "no-inner-declarations" rule in any engine:
+
+    <%= config.bin %> <%= command.id %> --rule-selector eslint:getter-return --rule-selector no-inner-declarations
+
+- Specify method starting points of the analysis:
+
+    <%= config.bin %> <%= command.id %> --path-start ./classes/Utils.cls#init --path-start ./classes/Helpers.cls#method1;method2
 
 # flags.workspace.summary
 
-*DRAFT*: Flag summary
+Set of files to be included in the code analysis. You can specify specific files or directories, and use glob patterns (wildcards). 
+
+# flags.workspace.description
+
+For most engines, a workspace is the set of files that are directly scanned for violations. For data flow analysis based engines, a workspace is the set the files that are included when the engines builds the control flow graph associated with the analysis. We recommended that you supply a workspace that contains all of your source files when selecting rules from data flow analysis based engines to ensure that the graph is complete. 
```

Comment:
A few notes:
1. Should probably be "data-flow-analysis-based?"
2. Even in DFA, `--workspace` is still by default the set of files directly scanned. If you want to filter that, then you use `--path-start`.